src/main/java/com/sprint/mission/discodeit/service/basic/BasicReadStatusService.java

```diff
@@ -23,8 +23,9 @@ public class BasicReadStatusService implements ReadStatusService {
     private final UserRepository userRepository;
 
     @Override
-    public ReadStatus create(ReadStatusCreateDTO readStatusCreateDTO, boolean isChannelExist) {
-        if(!isChannelExist &&  !userRepository.existByUserId(readStatusCreateDTO.userId())){
+    public ReadStatus create(ReadStatusCreateDTO readStatusCreateDTO) {
+        if(!readStatusRepository.existByChannelId(readStatusCreateDTO.channelId())
+            &&  !userRepository.existByUserId(readStatusCreateDTO.userId())){
            throw new IllegalArgumentException("ReadStatus를 생성할 수 없습니다.");
         }
```

Comment:
if문을 분리하더라도 에러 메시지를 상세하게 적어주는게 좋을 것 같아요.  

```suggestion
        if (!readStatusRepository.existByChannelId(readStatusCreateDTO.channelId())) {
            throw new IllegalArgumentException(String.format("존재하지 않는 채널이므로 ReadStatus를 생성할 수 없습니다. %s", readStatusCreateDTO.channelId()));
        }

        if (!userRepository.existByUserId(readStatusCreateDTO.userId())) {
            throw new IllegalArgumentException(String.format("존재하지 않는 사용자이므로 ReadStatus를 생성할 수 없습니다. %s", readStatusCreateDTO.userId()));
        }
```  


추가로 에러메시지를 찍을 땐 ID도 같이 남기는게 나중에 에러를 인지할 때 더 좋아요!